src/.env.sample

```diff
@@ -120,28 +120,39 @@ ALLOWED_HOST=http://exampleDomain.com
 AUTH_METHOD=jwt_only #or user_service_authenticated
 
 # instance level configuration regarding review flow 
-REVIEW_REQUIRED = true
+REVIEW_REQUIRED=true
 
 # instance level configuration regarding reviewer list
-SHOW_REVIEWER_LIST =true
+SHOW_REVIEWER_LIST=true
 
 # instance level configuration regarding review flow number of minimum approvals required.
 MIN_APPROVAL=1
 
 # instance level configuration regarding review type can be  SEQUENTIAL / PARALLEL
-REVIEW_TYPE = 'SEQUENTIAL'
+REVIEW_TYPE=SEQUENTIAL
 
 # mention the organization read API to the curresponding service 
-ORGANIZATION_READ_ENDPOINT = 'v1/organization/read'
+ORGANIZATION_READ_ENDPOINT=v1/organization/read
 
 # mention the user list API to the curresponding service 
-USER_LIST_ENDPOINT='v1/account/search'
+USER_LIST_ENDPOINT=v1/account/search
 
 # mention the user read API to the curresponding service 
-USER_PROFILE_DETAILS_ENDPOINT='v1/user/read'
+USER_PROFILE_DETAILS_ENDPOINT=v1/user/read
 
 #Release 1.0
 #available resource types
 RESOURCE_TYPES="projects,observations,observation_with_rubrics,surveys,programs"
 #maximum number of tasks allowed in a project
 MAX_PROJECT_TASK_COUNT= 10
+
+#consumption portal name ex: self/diksha/elevate-project
+CONSUMPTION_SERVICE=self
+
+#Publish method to consumption side kafka/api
+PUBLISH_METHOD=kafka
+
+#kafka topic for publish
+PROJECT_PUBLISH_KAFKA_TOPIC=publishtopic
```

Comment:
can we make it resource publish kafka topic?  @priyanka-TL

Reply:
for different resources publish topic will be different, like elevate-project can create the project, but if we want to create observation then we need to call elevate-samiksha kafka topic

Reply:
ohk got it @priyanka-TL